### DISCUSSION TITLE
Read system temp as sensor

### DISCUSSION THREAD

**Discussion by blatogera** [theyosh/TerrariumPI#711] [Q&A] [answered]
Hello, 
can you help how to add system temperature as a remote sensor? 
there was a workaround on v3 http://localhost:8090/api/system#temperature but it doesnt work on v4 for me

Thank you

**Comment by theyosh** [comment, marked as answer]
Hi,

I think this should work: http://localhost:8090/api/system_status/#cpu_temperature

Regards

**Reply by blatogera** [reply]
Hi,
it works! 
thank you!
